client.py

```diff
@@ -0,0 +1,44 @@
+import argparse
+import sys
+import socket
+import struct
+
+
+
+def send_data(server_ip, server_port, data):
+    client = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    client.connect((server_ip, server_port))
+    data_len=len(data)
+    data_len=struct.pack('<I', data_len)
+    client.sendall(data_len)
+    client.sendall(data.encode())
+    client.close()
+
+
+
+def get_args():
+    parser = argparse.ArgumentParser(description='Send data to server.')
+    parser.add_argument('server_ip', type=str,
+                        help='the server\'s ip')
```

Comment:
Write strings inside "" and then there is no need for escaping ' inside the string.